mitosheet/mitosheet/tests/dash/test_dash.py

```diff
@@ -0,0 +1,99 @@
+import pytest
+from dash import Input, Output, State, Dash, html, dcc
+from mitosheet.tests.decorators import requires_dash
+from mitosheet.mito_dash.v1 import Spreadsheet, mito_callback
+
+
+@requires_dash
+def test_can_create_dash_spreadsheet_component():
+    s = Spreadsheet(id='an id')
+    assert s is not None
+
+
+@requires_dash
+def test_mito_callback_component_callable_with_no_mito():
+    assert callable(mito_callback)
+
+    # Make a dash app
+    app = Dash(__name__)
+
+    app.layout = html.Div([
+        dcc.Input(id='input', value='initial value'),
+        html.Div(id='output'),
+        dcc.Store(id='state'),
+    ])
+    
+    @mito_callback(
+        Output('output', 'value'),
+        Input('input', 'value'),
+        State('state', 'value'),
+    )
```

Comment:
Can you point me in the right direction of where I should learn about this stuff. Is there any reason you would want this without Mito?

Reply:
See the dash docs, as well as the app I asked you to build in Zulip.